lib/core/src/text/buffer/line.rs

```diff
@@ -0,0 +1,156 @@
+use crate::prelude::*;
+
+use crate::text::buffer::glyph_square::
+{GlyphAttributeBuilder,GlyphVertexPositionBuilder,GlyphTextureCoordinatesBuilder};
+
+
+// ============================
+// === LineAttributeBuilder ===
+// ============================
+
+/// Buffer data for line of text builder
+///
+/// This builder makes a fixed-size buffers for a specific attribute (e.g. vertex position or
+/// texture coordinates) for one line. If line is longer than `max_line_size`, it is
+/// cut. When line is shorter, the buffer is padded with empty values (obtained from
+/// `GlyphAttributeBuilder::empty()`).
+pub struct LineAttributeBuilder<'a,GlyphBuilder:GlyphAttributeBuilder> {
+    max_line_size    : usize,
+    squares_produced : usize,
+    glyph_builder    : GlyphBuilder,
+    chars_iterator   : Option<std::str::Chars<'a>>,
+}
+
+pub type
+LineVerticesBuilder<'a,'b>           = LineAttributeBuilder<'a,GlyphVertexPositionBuilder<'b>>;
+pub type
+LineTextureCoordinatesBuilder<'a,'b> = LineAttributeBuilder<'a,GlyphTextureCoordinatesBuilder<'b>>;
```

Comment:
If we cannot change the name to be shorter (andplease do if we can somehow), I would definitely write it this way:
```rust
pub type LineVerticesBuilder<'a,'b> = 
    LineAttributeBuilder<'a,GlyphVertexPositionBuilder<'b>>;
pub type LineTextureCoordinatesBuilder<'a,'b> = 
    LineAttributeBuilder<'a,GlyphTextureCoordinatesBuilder<'b>>;
```

Reply:
Usually I'm against shorting names just because they don't fit line. And I'm usually against abbreviations. But then I discover that `GlyphTextureCoordsBuilder` is better than `GlyphTextureCoordinatesBuilder` :D 

So, each rule has exceptions :)